src/js/home.js

```diff
@@ -155,10 +155,10 @@ const AddForm = {
         main.appendChild(closedQuestion);
     },
 
-    addNewQuestion(id, isOpen) {
+    addNewQuestion(className, isOpen) {
         const question = document.createElement('input');
         question.classList.add('question');
-        question.id = id;
+        question.className = className;
```

Comment:
To się wyklucza,
usuń to question.classList.add('question');
i swoje zmien na question.classList.add(className);